GVRf/Framework/framework/src/main/java/org/gearvrf/GVRJassimpAdapter.java

```diff
@@ -257,96 +260,99 @@ public GVRMesh createMesh(GVRContext ctx, AiMesh aiMesh, EnumSet<GVRImportSettin
                 }
             }
         }
-        // Bones
-        if (doAnimation && aiMesh.hasBones())
-        {
-            processBones(mesh, aiMesh.getBones());
-        }
         return mesh;
     }
 
     public void setMeshMorphComponent(GVRMesh mesh, GVRSceneObject sceneObject, AiMesh aiMesh)
     {
-
         int nAnimationMeshes = aiMesh.getAnimationMeshes().size();
-        if(nAnimationMeshes == 0)
+        if (nAnimationMeshes == 0)
             return;
 
-        GVRMeshMorph morph = new GVRMeshMorph(mContext, nAnimationMeshes);
-        sceneObject.attachComponent(morph);
-        int blendShapeNum = 0;
-
-        for(AiAnimMesh animMesh : aiMesh.getAnimationMeshes())
+        try
         {
-            GVRVertexBuffer animBuff = new GVRVertexBuffer(mesh.getVertexBuffer(),
-                    "float3 a_position float3 a_tangent float3 a_normal float3 a_bitangent");
-
-            float[] vertexArray = null;
-            float[] normalArray = null;
-            float[] tangentArray = null;
-            float[] bitangentArray = null;
+            GVRMeshMorph morph = new GVRMeshMorph(mContext, nAnimationMeshes);
+            sceneObject.attachComponent(morph);
+            int blendShapeNum = 0;
 
-            //copy target positions to anim vertex buffer
-            FloatBuffer animPositionBuffer = animMesh.getPositionBuffer();
-            if (animPositionBuffer != null) {
-                vertexArray = new float[animPositionBuffer.capacity()];
-                animPositionBuffer.get(vertexArray, 0, animPositionBuffer.capacity());
-            }
-            animBuff.setFloatArray("a_position",vertexArray);
+            for (AiAnimMesh animMesh : aiMesh.getAnimationMeshes())
```

Comment:
This block (loop+.update()) is much slower in this PR.

Prior gvr-morph spent 0.8 secs here.
With this PR this takes 54 secs, 17 secs of which for update().

Reply:
But the code is the same. Maybe the content has changed? I will investigate.

Reply:
Correction. update is fast both pre and post this PR.
The loop iterations are 75 in both cases.

Before this PR an iteration takes 1ms (4 outliers at 10ms).
After this PR every iteration takes about half-a-second.